src/images/mozilla-logo.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 96">

```

Comment:
The logo at https://mozilla.ninja/our-logo/ seems to have more whitespace:

![](https://static1.squarespace.com/static/58879499725e2572b9c4e0ec/t/5889828d15d5dbf0ba46d43d/1485406868486/?format=2500w)